.storybook/blocks/TabContainer/TabContainer.tsx

```diff
@@ -0,0 +1,28 @@
+import React, { ReactNode, useState } from 'react';
+
+import styles from './TabContainer.module.css';
+
+type Tab = {
+  label: string;
+  component: ReactNode;
+};
+
+type TabContainerProps = {
+  tabs: Tab[];
+};
```

Comment:
I know we do not have now the code guidelines. 
I'll create it.
But, for now, let's move the type and interfaces to a separate file `TabContainer.types.ts`